src/BaseTemplates/StarterWeb/Startup.cs

```diff
@@ -39,8 +39,12 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             if (env.IsDevelopment())
             {
-                app.UseDeveloperExceptionPage();
-                app.UseBrowserLink();
+                app.UseDeveloperExceptionPage();$if$ ($platformversion$ == 1.1)
```

Comment:
Should this be >= 1.1. IF the version is 1.1.1 then the condition would be false and it will still enable BrowserLink, no?